articles/azure/azs-gs.md

```diff
@@ -44,9 +44,88 @@ The two main delivery models provided by UKCloud for Microsoft Azure are:
 - A multi-tenant IaaS and PaaS platform available in both UKCloud security domains (Assured OFFICIAL and Elevated OFFICIAL), connected to government community networks with UKCloud's award-winning support
 - Private cloud deployment of UKCloud for Microsoft Azure dedicated to a single customer
 
-## Before you begin
+### What is Azure Active Directory and why is it required for UKCloud for Microsoft Azure?
 
-You should have received your UKCloud for Microsoft Azure login credentials from your Azure administrator. If you're the administrator, your Customer Success Manager (CSM) will have provided you with your login credentials.
+Azure Active Directory is Microsoft's cloud-based Directory and Identity Access Management Service. It provides Single Sign-On and core directory services via a high availability service with automated failover. <abbr title="Azure Active Directory"> AAD</abbr> provides the identity store for UKCloud for Microsoft Azure, supporting features such as two-factor authentication and role based access control.
+
+> [!Note]
+> If you do not have an existing Azure Active Directory, please provide a preferred domain name and administrator email address, and UKCloud will create an Azure Active Directory Domain (subject to availability) free of charge.
+
+## How can I get started with UKCloud for Microsoft Azure?
+
+# [I have an existing Azure Active Directory Domain](#tab/tabid-1)
+
+If you have decided that UKCloud for Microsoft Azure is the right environment for you, please contact your CSM who will request the following information:
```

Comment:
I think we could possibly simplify this a bit. Something like:

If you've decided that UKCloud for Microsoft Azure is the right environment for you, contact you CSM with the following information:

Contact email address, for example, onboardingukcloud3@ukcloud.com
Azure Active Director domain, for example, onboardingukcloud3csp.onmicrosoft.com
AAD administrator email address, for example, admin@onboardingukcloud3csp.onmicrosoft.com

(And make sure that the email addresses, etc above are not real and are safe to share publicly)